### DISCUSSION TITLE
RPi Docker

### DISCUSSION THREAD

**Discussion by joel-madero** [mealie-recipes/mealie#2043] [General]
Hi All -
I had Mealie working flawlessly on my RPi but decided to start from scratch and now I'm hitting an error when I try to install the docker. Has something changed?

`joel@raspberrypi:~ $ docker run \
    -p 9925:80 \
    -v `pwd`:'/app/data/' \
    hkotel/mealie:latest
Unable to find image 'hkotel/mealie:latest' locally
latest: Pulling from hkotel/mealie
docker: no matching manifest for linux/arm/v7 in the manifest list entries.
See 'docker run --help'.
`

**Comment by michael-genson** [comment]
arm/v7 builds are no longer supported. Some users have said Mealie works great on the 64bit OS. More info here: https://github.com/mealie-recipes/mealie/discussions/2283
